src/main/java/com/example/gonggang/admin/domain/Admin.java

```diff
@@ -0,0 +1,51 @@
+package com.example.gonggang.admin.domain;
+
+import com.example.gonggang.domain.users.domain.Role;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity(name = "admin")
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Admin {
+
+	@Column(nullable = false)
+	Role role;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long Id;
```

Comment:
```suggestion
@Id
@GeneratedValue(strategy = GenerationType.IDENTITY)
private Long Id;

@Column(nullable = false)
Role role;
```
이런식으로 id를 위로 올리는게 좋아보입니다

Reply:
이부분을 놓치고 있었네요! 수정했습니다!